### PR TITLE
Added columns period start and end

### PR DESCRIPTION
**Pull request by lijol**
Pull Request
**Are you a current Fivetran customer?** 

Lijo Lawrance
Data Engineer
Apollo.io

**What change(s) does this PR introduce?** 
Included the column period_start and period_end to stg_stripe__invoice table

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  The PR adds two columns to stripe invoice table

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature [#50 ]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [X] Local (please provide additional testing details below)
image
image

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [X] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/stg_stripe__invoice.sql [hunk 1]
```diff
@@ -43,8 +43,9 @@ final as (
         tax,
         tax_percent,
         total,
-        subscription_id
-        
+        subscription_id,
+        period_start,
+        period_end
```
Comment (fivetran-joemarkiewicz) [inline models/stg_stripe__invoice.sql:48]: Would you be able to add these fields and descriptions to the invoice tables in the `stg_stripe.yml` and `src_stripe.yml` files.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# dbt_stripe_source v0.7.3
+
+## 🎉 Feature Updates
+- Two columns added to stg_stripe__invoice (period_start, period_end)🧱 ([#50](https://github.com/fivetran/dbt_stripe_source/pull/51))
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:4]: We probably don't need to include the 🧱 in this CHANGELOG entry.
```suggestion
- Two columns added to stg_stripe__invoice (period_start, period_end) ([#50](https://github.com/fivetran/dbt_stripe_source/pull/51))
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@lijol thanks so much for taking the time to open this PR! We really value your contribution to the Fivetran dbt package codebase 🎉. I have just a few small requested changes. Once those are applied we can look to approving and merging into the next release.

File: dbt_project.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'stripe_source'
-version: '0.7.2'
+version: '0.7.3'
```
[the remark this answers is not in the document]
Comment (lijol) [reply dbt_project.yml:4]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+
```
[the remark this answers is not in the document]
Comment (lijol) [reply CHANGELOG.md:7]: Added the same

File: models/stg_stripe__invoice.sql (hunk 1 above)
Comment (lijol) [reply models/stg_stripe__invoice.sql:48]: Added to both the files

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@lijol thanks so much for applying the updates and for contributing to our dbt package! I made a few small edits, but all looks good on my end!

**Pull request merged**